django_geosource/tests/test_models.py

```diff
@@ -247,3 +249,21 @@ def test_get_status(self):
 
     def test_refresh_data(self):
         self.assertEqual({}, self.source.refresh_data())
+
+
+class ModelCSVSourceTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.source_name = os.path.join(
+            settings.BASE_DIR, "django_geosource", "tests", "source.csv"
+        )
+        cls.source = CSVSource.objects.create(
+            file=cls.source_name,
+            geom_type=8,
+            longitude_field=42.2222,
```

Comment:
But why ????

Reply:
Changes/rework have been made to reflect more the functional needs.